Examples/HttpAnalyzer/main.cpp

```diff
@@ -280,24 +251,20 @@ void printHostnames(HttpRequestStats& reqStatscollector)
 /**
  * Print the status code count table
  */
-void printStatusCodes(HttpResponseStats& resStatscollector)
+void printStatusCodes(const HttpResponseStats& resStatscollector)
 {
 	// create the table
-	std::vector<std::string> columnNames;
-	columnNames.push_back("Status Code");
-	columnNames.push_back("Count");
-	std::vector<int> columnsWidths;
-	columnsWidths.push_back(28);
-	columnsWidths.push_back(5);
+	std::vector<std::string> columnNames = {"Status Code", "Count"};
+	std::vector<int> columnsWidths = {28, 5};
 	pcpp::TablePrinter printer(columnNames, columnsWidths);
 
-	// go over the status code map and print each item
```

Comment:
@seladb This sections, when compared with the older ones, seem not to benefit that much from the use of range base loops. The main reason: the need to sort the collections and having to code custom comparators keeps the code unreadable. I tried using std::ranges and this, in my opinion, makes it readable... but that is not C++11.

Reply:
I think the code changes you made are ok 👍